### PR TITLE
Replaces JS styleguide with XO + Prettier configuration

### PR DESCRIPTION
**Pull request by larimaza**
# What 
Replace JS styleguide description with XO + Prettier configuration.

# Why 
We decided to replace our descriptive styleguide with just one automated decision making system, in order to improve our productivity while also enforcing more strict code standards.

# How 
Setup XO and Prettier (linter and formatter). Formatting rules from Prettier precede XO ones.
Remove previous styleguide descriptions and examples.

### CODE REVIEW

**Comment by larimaza** [comment]
[💻: 2, {Documentação} Definir estilo de código para JS](https://trello.com/c/iQT3hsvS/6814-%F0%9F%92%BB-2-documenta%C3%A7%C3%A3o-definir-estilo-de-c%C3%B3digo-para-js)

File: JavaScript.md [hunk 1]
```diff
@@ -2,1105 +2,22 @@
...
-* [Links](#links)
-  * [Styleguides](#styleguides)
-  * [Articles and blog posts](#articles-and-blog-posts)
-  * [Tools](#tools)
+## Rules
+We adopt all rules from XO and Prettier, but formatting rules from Prettier have precedence over XO ones. Both tools know about eachother so they'll never conflict.
 
-## Disclaimer
+## Config
+An example Prettier configuration file can be found [here](/linters/js/.prettierrc), which will be used by both Prettier and XO. The dependencies and XO configuration can be found [here]().
```
Comment (luisfmsouza) [inline JavaScript.md]: Link missing in the end of this line.

Comment (larimaza) [reply JavaScript.md]: 002063

Comment (larimaza) [reply JavaScript.md]: 002063

Comment (larimaza) [reply JavaScript.md]: 0020631

**Pull request merged**